cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java

```diff
@@ -581,12 +581,21 @@ public void upsertTransaction (LinkedHashMap<String, ArrayList<JsonElement>> agg
             String insertQuery = SQLQueryUtils.sqlInsertQuery(aggregation,
                     tableName,
                     sqlInstance,
-                    destination).toString();
+                    destination,
+                    attrNativeTypes).toString();
 
-            PreparedStatement insertStatement = null;
+            PreparedStatement insertStatement;
             insertStatement = connection.prepareStatement(insertQuery);
+            /*
+
+            FIXME
```

Comment:
Maybe this FIXME should mention an issue number? Is this related with #1959?

Reply:
(Same in the other FIXME below)

Reply:
Fixed in (6405c46)[https://github.com/telefonicaid/fiware-cygnus/pull/1960/commits/6405c4618697b38182c45733d38268d817e903fd]